float32 panSpeed
float32 tiltSpeed
float32 zoomSpeed
uint32 timeoutMs
---
int32 status